File: test/MapperAI.Test/Class/Student.py
class Student:
    def __init__(self, name, age, student_id):
        self.name = name
        self.age = age
        self.student_id = student_id

    def get_name(self):
        return self.name

    def get_age(self):
        return self.age

    def get_student_id(self):
        return self.student_id

    def is_adult(self):
        return self.age >= 18

File: test/MapperAI.Test/MappedClasses/Student.js
namespace MapperAI.Test.MappedClasses {
  export class Student {
    private name: string;
    private age: number;
    private student_id: string;

    constructor(name: string, age: number, student_id: string) {
      this.name = name;
      this.age = age;
      this.student_id = student_id;
    }

    public getName(): string {
      return this.name;
    }

    public getAge(): number {
      return this.age;
    }

    public getStudentId(): string {
      return this.student_id;
    }

    public isAdult(): boolean {
      return this.age >= 18;
    }
  }
}
